src/notation/qml/MuseScore/NotationScene/qmldir

```diff
@@ -9,3 +9,4 @@ ConcertPitchControl 1.0 ConcertPitchControl.qml
 NoteInputBar 1.0 NoteInputBar.qml
 NoteInputBarCustomizationDialog 1.0 NoteInputBarCustomizationDialog.qml
 UndoRedoToolBar 1.0 UndoRedoToolBar.qml
+ExportDialog 1.0 ExportDialog.qml
```

Comment:
you must export dialogs in qmldir of the module where you stored the file(in your case it is UserScore)